src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition2Test.java

```diff
@@ -277,4 +277,12 @@ public void rhsOfDeclarationSandboxedInCPS() throws Exception {
         jenkins.assertBuildStatus(Result.FAILURE, b);
         jenkins.assertLogContains("org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use staticMethod jenkins.model.Jenkins getInstance", b);
     }
+
+    @Issue("JENKINS-47064")
+    @Test
+    public void booleanClosureWrapperFromDGM() throws Exception {
+        WorkflowJob job = jenkins.jenkins.createProject(WorkflowJob.class, "p");
+        job.setDefinition(new CpsFlowDefinition("['a', 'b'].every { sleep 1; return it != null }\n", true));
```

Comment:
Maybe you also want to assert the return value of `every`?

Reply:
Wouldn't hurt - we've already got tests over in `CpsDefaultGroovyMethodsTest` that verify the expected behavior without the `sleep`, but hey.